### ISSUE TITLE
Missing a GitHub issue form template for docs issues

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [withastro/starlight#2583]
### What version of `starlight` are you using?

n/a

### What version of `astro` are you using?

n/a

### What package manager are you using?

n/a

### What operating system are you using?

n/a

### What browser are you using?

n/a

### Describe the Bug

I'd like to suggest some additions to docs, but the only template I can use is 🐛 Bug Report. Could we have a template for https://starlight.astro.build docs suggestions, please?

Filing as a bug report issue as suggested on Discord: https://discord.com/channels/830184174198718474/1070481941863878697/1303808425943236608

### Link to Minimal Reproducible Example

n/a

### Participation

- [X] I am willing to submit a pull request for this issue.

**Comment by delucis** [comment]
> Could we have a template for [starlight.astro.build](https://starlight.astro.build) docs suggestions, please?

Sure! Seems like a good idea to me.

Although I might suggest a distinction based on how we use the Starlight bug tracker:

1. An issue template for errors/problems with docs
2. A _discussion_ template for new docs content suggestions

So I guess that’s two new things 😁 

Basically, a while back we moved to using GitHub issues exclusively for bug reports, moving [feature requests](https://github.com/withastro/starlight/discussions/categories/feature-requests) to discussions and I love having that distinction. A feature request might be a good idea but still take 2 years to get to sometimes and I found having those long-lived ideas gathering feedback alongside more urgent bug reports was difficult to manage. It lets us have a really clear “bugs” inbox of stuff to fix while still letting people share ideas that may take longer to get to (or require more community feedback).

So I think a similar distinction between “SOMETHING IS WRONG IN DOCS!” and “Hey, I would have loved to have a guide about X” would be good here too.

Are you interested in making a PR proposing these templates to kick things off?
